### PR TITLE
Add more a profile with scrolly thingy

### LINKED ISSUE #52: Make profile activity a scroll activity
The profile information should be scrolled away because it can be long and it can clog the repository list.

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Resolve #52

### CODE REVIEW

**Comment by McSinyx** [comment]
The following uncrashes it (regarding colorSurface, you need to define it in a theme (?)), although I doubt it's working as intended ATM:

```diff
diff --git a/app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java b/app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java
index 3bf3e7f..75eb09a 100644
--- a/app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java
+++ b/app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java
@@ -40,9 +40,9 @@ public class ProfileActivity extends BaseActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-      
-        TextView txt_toolbar = (TextView) findViewById(R.id.main_text_bar);
-        txt_toolbar.setText("Profile");
+        // id/main_text_bar is not defined anywhere
+        //TextView txt_toolbar = (TextView) findViewById(R.id.main_text_bar);
+        //txt_toolbar.setText("Profile");
         LinearLayout layout = this.findViewById(R.id.profile);
         layout.setBackgroundColor(getApplicationContext().getResources().getColor(R.color.secondaryColor));
       
diff --git a/app/src/main/res/layout/activity_profile_alt.xml b/app/src/main/res/layout/activity_profile_alt.xml
index d455c18..7f65889 100644
--- a/app/src/main/res/layout/activity_profile_alt.xml
+++ b/app/src/main/res/layout/activity_profile_alt.xml
@@ -40,9 +40,7 @@ android:id="@+id/drawer_layout">
             android:id="@+id/collapsingToolbar"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            app:contentScrim="?attr/colorSurface"
             app:layout_scrollFlags="scroll|exitUntilCollapsed"
-            app:statusBarScrim="?attr/colorSurface"
             app:titleEnabled="false">
 
             <!-- Profile header: profile picture and names -->
@@ -51,7 +49,7 @@ android:id="@+id/drawer_layout">
                 android:layout_height="110dp"
                 android:orientation="horizontal">
 
-                <com.google.android.material.imageview.ShapableImageView
+                <com.google.android.material.imageview.ShapeableImageView
                     android:id="@+id/profile_picture"
                     android:layout_width="80dp"
                     android:layout_height="80dp"
```

**Comment by Huy-Ngo** [comment]
Thanks. It worked, but there's a long way to go.
Screenshot_2020-12-27-10-06-03-395_vn edu usth minigh

**Comment by Huy-Ngo** [comment]
I don't understand why the toolbar title doesn't show.

File: app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java [hunk 1]
```diff
@@ -33,16 +33,14 @@
...
+        super(R.layout.activity_profile_alt);
     }
 
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
       
-        TextView txt_toolbar = (TextView) findViewById(R.id.main_text_bar);
-        txt_toolbar.setText("Profile");
```
Comment (McSinyx) [inline app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java:45]: You meant this one?

Comment (Huy-Ngo) [reply app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java:45]: Now I get another problem: The title shows even when it's expanded.

**Comment by Huy-Ngo** [comment]
It works now.

**Comment by McSinyx** [comment]
Can you make it work for scroll event (not sure what it's called, I mean the one with mouse buttons 4 and 5)?  Also please bring back the button to access the drawer.

**Comment by Huy-Ngo** [comment]
> Can you make it work for scroll event (not sure what it's called, I mean the one with mouse buttons 4 and 5)?

I don't get what you mean. Isn't it already working for scrolling?

> bring back the button to access the drawer

I'll try

**Comment by McSinyx** [comment]
It works with swiping gestures, not with scrolling (on phones most of the times it doesn't matter until one plugs in a mouse though, hence the *can you* question).

**Comment by Huy-Ngo** [comment]
> It works with swiping gestures, not with scrolling

I don't know how to emulate that.

**Review by McSinyx — APPROVED** [review]
This is not perfect, but I supposed it can be fixed by a follow-up PR.

2020-12-29T10:33:07

**Review by McSinyx — APPROVED** [review]
All good now!

**Pull request merged**